client/js/src/upload.ts

```diff
@@ -18,46 +18,18 @@ export function normalise_file(
 	proxy_url: string | null // root_url: string | null
 ): FileData[] | FileData | null;
 
+/*
```

Comment:
Is there a need for backwards compatibility? wouldn't previous custom components be version pinned for the client?

Reply:
They should but we would need to make the client version bump minor, to respect semver.

Reply:
(Minor because this is a pre-1.0 version, so minor is breaking).

Reply:
Good point @aliabid94 @pngwn I'll change

Reply:
Done